sbtc/src/testing/emily.rs

```diff
@@ -0,0 +1,258 @@
+        let gsis = parse_gsis(gsi_array);
+        builder = builder.set_global_secondary_indexes(Some(gsis));
+    }
+
+    builder.send().await.expect("failed to create table");
+}
+
+fn parse_key_schema(schema_json: &serde_json::Value) -> Vec<KeySchemaElement> {
+    let mut keys = Vec::new();
+    for key in schema_json.as_array().unwrap() {
+        let name = key["AttributeName"].as_str().unwrap();
+        let type_str = key["KeyType"].as_str().unwrap();
+
+        let key_type = match type_str {
+            "HASH" => KeyType::Hash,
+            "RANGE" => KeyType::Range,
+            _ => panic!("unknown key type: {type_str}"),
+        };
+
+        keys.push(
+            KeySchemaElement::builder()
+                .attribute_name(name)
+                .key_type(key_type)
+                .build()
+                .unwrap(),
+        );
+    }
+    keys
+}
+
+fn parse_gsis(gsi_array: &Vec<serde_json::Value>) -> Vec<GlobalSecondaryIndex> {
+    let mut indexes = Vec::new();
+
+    for gsi in gsi_array {
+        let index_name = gsi["IndexName"].as_str().unwrap();
+        let key_schema = parse_key_schema(&gsi["KeySchema"]);
+
+        let proj_json = &gsi["Projection"];
+        let proj_type_str = proj_json["ProjectionType"].as_str().unwrap();
+
+        let mut proj_builder = Projection::builder();
+
+        let proj_type = match proj_type_str {
+            "ALL" => ProjectionType::All,
+            "KEYS_ONLY" => ProjectionType::KeysOnly,
+            "INCLUDE" => ProjectionType::Include,
+            _ => panic!("unknown projection type: {proj_type_str}"),
+        };
+        proj_builder = proj_builder.projection_type(proj_type);
+
+        if proj_type_str == "INCLUDE"
+            && let Some(attrs) = proj_json["NonKeyAttributes"].as_array()
+        {
+            let attr_names: Vec<String> = attrs
+                .iter()
+                .map(|v| v.as_str().unwrap().to_string())
+                .collect();
+            proj_builder = proj_builder.set_non_key_attributes(Some(attr_names));
+        }
+
+        let gsi_builder = GlobalSecondaryIndex::builder()
+            .index_name(index_name)
+            .set_key_schema(Some(key_schema))
+            .set_projection(Some(proj_builder.build()));
+
+        indexes.push(gsi_builder.build().unwrap());
+    }
+
+    indexes
+}
+
+async fn delete_table(client: &Client, table_name: &str) {
+    client
+        .delete_table()
+        .table_name(table_name)
+        .send()
+        .await
+        .unwrap();
+}
+
+fn get_test_client() -> Client {
+    let creds = aws_sdk_dynamodb::config::Credentials::new(
+        "xxxxxxxxxxxx",
+        "xxxxxxxxxxxx",
+        None,
+        None,
+        "test",
+    );
+
+    // We don't need https in tests (and we avoid wasting time in loading certs)
+    let tcp_connector = hyper_014::client::HttpConnector::new();
+    let http_client = HyperClientBuilder::new().build(tcp_connector);
```

Comment:
Using the default client takes ~150ms, mostly to load the certificates (that we don't use for local tests), and since nextest uses a process for each test we cannot just rely on cached clients :(